api/src/pcapi/scripts/validate_collective_offer_offererAddressId_fk_constraint/main.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import text
+
+from pcapi import settings
+from pcapi.app import app
+from pcapi.models import db
+
+
+app.app_context().push()
+
+
+def validate_constraint() -> None:
+    # Disabling lock_timeout & statement_timeout as we can't know in advance how long it would take
+    db.session.execute(text("SET lock_timeout = 0;"))
```

Comment:
Très mauvaise idée, ça peut créer une interruption de service.

Reply:
Pourquoi donc ?

Reply:
vu ensemble, j'enlève la désactivation du lock_timeout, d'après la doc postgres "This mode protects a table against concurrent schema changes and VACUUM runs." donc à priori ça ne rentrera pas en conflit ni avec la lecture ni l'écriture dans la table, donc pas de raison que l'on lock timeout